src/server/_params.py

```diff
@@ -0,0 +1,133 @@
+import re
+from dataclasses import dataclass
+from typing import Any, List, Optional, Sequence, Tuple, Union
+
+from flask import request
+
+from ._exceptions import ValidationFailedException
+
+
+def _parse_common_multi_arg(key: str) -> List[Tuple[str, Union[bool, Sequence[str]]]]:
+    line = ";".join(request.values.getlist(key))
```

Comment:
such that `?time=day:20201010&time=day:20201212` variant is supported

Reply:
right but why not just 
```
for entry in request.values.getlist(key):
```
instead of the join+split?

Reply:
cause users can mix it both variants, e.g. `?time=day:20201010;day:20201011&time=day:20201212` -> so three entries in total. I tried to avoid another nested loop in this case and merge them to create one unified parameter.